Saas 817 list of support tickets UI tests

peterSirotnak opened the pull request:
Pull request for [SAAS-817](https://jira.percona.com/browse/SAAS-817)

yurkovychv commented on cypress/integration/customer/dashboard/dashboard.spec.js:
```diff
@@ -26,4 +27,50 @@ context('Dashboard Tests for customers', () => {
+        .get('th')
+        .then((cells) => {
+          expect(cells.length).to.be.equal(7);
+          for (let i = 0; i < cells.length; i++) {
```
it's better to use [expect](https://docs.cypress.io/guides/references/assertions#BDD-Assertions) here too. Example: `expect([1, 2, 3]).to.include.members([3, 2])`.

peterSirotnak replied:
Okay, fixed.

yurkovychv commented on cypress/integration/customer/dashboard/helper.js:
```diff
@@ -8,3 +10,8 @@ export const createOrgAndAddUsers = (users) => {
   );
 };
+
+export const isTicketTableHeaderTextCorrect = (cellText, position) => {
```
I believe this one is not needed if you will use `expect` from comment above

peterSirotnak replied:
Yes that is correct, removed.

yurkovychv commented:
@peterSirotnak Also please add `non-SN users that are members of an organization linked with SN` test

peterSirotnak commented:
> @peterSirotnak Also please add `non-SN users that are members of an organization linked with SN` test. And test case IDs to test summary

Added that test and also added waiting for dashboard page to fully load. Ticket table is rendered after loading spinners are not present.

peterSirotnak commented on cypress/integration/customer/dashboard/dashboard.spec.js:
```diff
@@ -26,4 +29,65 @@ context('Dashboard Tests for customers', () => {
     });
   });
+
+  it('Verify Percona Customer user is able to see tickets created for his org', () => {
```
Fixed
